package.json

```diff
@@ -107,7 +107,7 @@
     "typescript": "^4.9.4"
   },
   "dependencies": {
-    "@tableland/evm": "^4.1.0",
+    "@tableland/evm": "^4.2.0-pre.0",
```

Comment:
After the contract upgrades happen, we can publish `@tableland/evm` 4.2.0 and update this value.